adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderService.java

```diff
@@ -1,17 +1,32 @@
 package com.microservicesteam.adele.ordermanager.domain;
 
+import java.math.BigDecimal;
 import java.time.LocalDateTime;
+import java.util.Currency;
 import java.util.function.Supplier;
 
 import org.springframework.stereotype.Service;
 
+import com.microservicesteam.adele.ordermanager.domain.exception.InvalidPaymentResponseException;
+import com.microservicesteam.adele.payment.PaymentManager;
+import com.microservicesteam.adele.payment.PaymentRequest;
+import com.microservicesteam.adele.payment.PaymentResponse;
+import com.microservicesteam.adele.payment.PaymentStatus;
+import com.microservicesteam.adele.payment.Ticket;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 @Service
 @RequiredArgsConstructor
 public class OrderService {
 
+    private static final String URL = "/orders/%s/payment?status=%s";
```

Comment:
The URL should contain http://domain as well as this is called by Paypal. 
We should import this parameter from config.